go/teams/handler.go

```diff
@@ -75,12 +88,42 @@ func HandleRotateRequest(ctx context.Context, g *libkb.GlobalContext, msg keybas
 	})
 }
 
+func HandleOpenTeamSweepRequest(ctx context.Context, g *libkb.GlobalContext, msg keybase1.TeamOpenSweepMsg) (err error) {
+	ctx = libkb.WithLogTag(ctx, "CLKR")
+	defer g.CTrace(ctx, fmt.Sprintf("HandleOpenTeamSweepRequest(teamID=%s,len(resetUsers)=%d)", msg.TeamID, len(msg.ResetUsersUntrusted)), func() error { return err })()
+
+	team, err := Load(ctx, g, keybase1.LoadTeamArg{
+		ID:          msg.TeamID,
+		Public:      msg.TeamID.IsPublic(),
+		ForceRepoll: true,
+	})
+	if err != nil {
+		return err
+	}
+
+	role, err := team.myRole(ctx)
+	if err != nil {
+		return err
+	}
+	if !role.IsOrAbove(keybase1.TeamRole_ADMIN) {
+		return fmt.Errorf("OpenSweep request for team %s but our role is: %s", team.ID, role.String())
+	}
+
+	// CanSkipKeyRotation() should return `true` for open teams, so sweeping
+	// will not rotate. But assume the possibility of OPENSWEEP being sent for
+	// non-open teams.
```

Comment:
I think it's confusing to have methods with "OpenTeam" in their name that are willing to operate on closed teams. How about the OpenTeam methods error out if the team is found to be closed?